script.js

```diff
@@ -0,0 +1,132 @@
+const computerChoiceIcon = document.getElementById("computerChoiceIcon");
+
+const play = () => {
+    rockBtn.addEventListener("click", () => {
+        let computerChoice = random();
+        computerChoiceSpan.innerText = computersOptions[computerChoice];
+        humanChoiceIcon.innerHTML =
+            '<i class="fa-solid fa-hand-back-fist fa-5x"></i>';
+        humanChoiceSpan.innerText = "Rock";
+        if (computerChoice === 0) {
+            result.innerText = "Draw";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-back-fist fa-5x"></i>';
+        } else if (computerChoice === 1) {
+            result.innerText = "Computer Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand fa-5x"></i>';
+            computerScore.innerText++;
+        } else {
+            result.innerText = "You Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+            humanScore.innerText++;
+        }
+    });
+    paperBtn.addEventListener("click", () => {
+        let computerChoice = random();
+        computerChoiceSpan.innerText = computersOptions[computerChoice];
+        humanChoiceIcon.innerHTML = '<i class="fa-solid fa-hand fa-5x"></i>';
+        humanChoiceSpan.innerText = "Paper";
+        if (computerChoice === 0) {
+            result.innerText = "You Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-back-fist fa-5x"></i>';
+            humanScore.innerText++;
+        } else if (computerChoice === 1) {
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand fa-5x"></i>';
+            result.innerText = "Draw";
+        } else {
+            result.innerText = "Computer Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+            computerScore.innerText++;
+        }
+    });
+    scissorBtn.addEventListener("click", () => {
+        let computerChoice = random();
+        computerChoiceSpan.innerText = computersOptions[computerChoice];
+        humanChoiceIcon.innerHTML =
+            '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+        humanChoiceSpan.innerText = "Scissor";
+        if (computerChoice === 0) {
+            result.innerText = "Computer Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-back-fist fa-5x"></i>';
+
+            computerScore.innerText++;
+        } else if (computerChoice === 1) {
+            result.innerText = "You Won!!!";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand fa-5x"></i>';
+            humanScore.innerText++;
+        } else {
+            result.innerText = "Draw";
+            computerChoiceIcon.innerHTML =
+                '<i class="fa-solid fa-hand-scissors fa-5x"></i>';
+        }
+    });
```

Comment:
for this specific case we can add logic of querySelectorAll("botton"), and for each button click we can call a method that will update the scenario who win.
Re- write the code: using concept of:
querySelectorAll("botton"), use foreach, and onclick call one method that will update the state, also avoid using multiple if...else and repeating code,  use ternary operator as user can win in three cases only
you_win = ( (you === "rock") && (opponent === "scissor") ||
(you === "paper") && (opponent === "rock") ||
(you === "scissor") && (opponent === "rock") ) // returns true for your win cases only.
so (you === opponent) ? "draw message" : you_win ? call a method for updating score and message for user : call method for updating score and message fro computer;